.azure-pipelines/template-stop-daemon.yml

```diff
@@ -0,0 +1,5 @@
+steps:
+- bash: |
```

Comment:
Is it possible to exit the daemon automatically?

Reply:
If we don't exit the daemon. Each job will start one daemon. It looks not secure.
cleanup is used in sonic-mgmt, sonic-slave and sonic-buildimage pipeline.